src/System.Drawing.Design.Primitives/tests/Design/ToolboxItemCollectionTests.cs

```diff
@@ -3,7 +3,8 @@
 // See the LICENSE file in the project root for more information.
 
 using Xunit;
-namespace System.Drawing.Design.Primitives.Tests.System.Drawing.Design
+
+namespace System.Drawing.Design
```

Comment:
Tests namespace should be `System.Drawing.Design.Tests`

Reply:
You are still missing the `.Tests` part in the namespace.